src/hosted-form/iframe-content/hosted-input-initializer.ts

```diff
@@ -54,6 +57,24 @@ export default class HostedInputInitializer {
             .toPromise();
     }
 
+    firefoxAndSafariBugfix(): void {
+        if (document.activeElement === document.body) {
+            const textField = document.querySelectorAll('input')[0];
```

Comment:
Could we move the event handler to `HostedInput`? That way we don't have to query for `input`, as it holds a direct reference to the input element. Also, we can hook up to the `attach/detach` methods.